handle-mulipart-form-data/backend_server/server.bal

```diff
@@ -0,0 +1,67 @@
+import ballerina/http;
+import ballerina/io;
+import ballerina/log;
+import ballerina/mime;
+import ballerina/random;
+
+type CustomerRegistrationData record {|
+    string firstName;
+    string lastName;
+    string address;
+    int dependents;
+|};
+
+final table<CustomerRegistrationData> customerTable = table [
+    {firstName: "James", lastName: "Clerk", address: "456, ElmAvenue, Suite-7, Willowville", dependents: 2},
+    {firstName: "John", lastName: "Doe", address: "789, OakLane, Unit-C12, Pineville", dependents: 6},
+    {firstName: "Anna", lastName: "Watson", address: "101, Maple-Road, OfficeD, Birchwood", dependents: 7}
+];
+
+@http:ServiceConfig {
+    cors: {
+        allowOrigins: ["*"]
+    }
+}
+service /crm on new http:Listener(9090) {
+    resource function get customers() returns CustomerRegistrationData[] {
+        return customerTable.toArray();
+    }
+
+    resource function get customers/[string customerId]/agreement() returns byte[]|http:NotFound {
+        byte[]|error agreementForm = getAgreementForm(customerId);
+        if agreementForm is error {
+            return <http:NotFound>{body: {message: "Agreement form not found for the customer ID: " + customerId}};
```

Comment:
If the file read fails we cannot return NoFound error. Better to return a generic error such as 500.

Reply:
Updated.